docs/syzbot.md

```diff
@@ -164,8 +164,20 @@ so please treat the results with understanding. You may consult the provided
 Suggestions and patches that improve bisection quality for common cases are
 [welcome](https://github.com/google/syzkaller/issues/1051).
 
+Syzbot supports cause bisection(find the commit that introduces a bug) and fix bisection(find the commit that fixes a bug).
```

Comment:
Few spelling fixes. We refer to syzbot as `syzbot` throughout the doc (even if it's the first word of a sentence. Please use the same for consistency.

Reply:
add spaces before `(`